code/modules/mob/living/simple_animal/humanoids/syndicate.dm

```diff
@@ -1,163 +1,192 @@
-			visible_message("\red \b [src] has been attacked with the [O] by [user]. ")
-		else
-			visible_message("\red \b [src] blocks the [O] with its shield! ")
-		//user.do_attack_animation(src)
-	else
-		usr << "\red This weapon is ineffective, it does no damage."
-		visible_message("\red [user] gently taps [src] with the [O]. ")
-
-
-/mob/living/simple_animal/hostile/syndicate/melee/bullet_act(var/obj/item/projectile/Proj)
-	if(!Proj)	return
-	if(prob(65))
-		src.health -= Proj.damage
-	else
-		visible_message("\red <B>[src] blocks [Proj] with its shield!</B>")
-	return 0
-
-
-/mob/living/simple_animal/hostile/syndicate/melee/space
-	min_oxy = 0
-	max_oxy = 0
-	min_tox = 0
-	max_tox = 0
-	min_co2 = 0
-	max_co2 = 0
-	min_n2 = 0
-	max_n2 = 0
-	minbodytemp = 0
-	icon_state = "syndicatemeleespace"
-	icon_living = "syndicatemeleespace"
-	name = "Syndicate Commando"
-	corpse = /obj/effect/landmark/mobcorpse/syndicatecommando
-	speed = 0
-
-/mob/living/simple_animal/hostile/syndicate/melee/space/Process_Spacemove(var/check_drift = 0)
-	return
-
-/mob/living/simple_animal/hostile/syndicate/ranged
-	ranged = 1
-	rapid = 1
-	icon_state = "syndicateranged"
-	icon_living = "syndicateranged"
-	casingtype = /obj/item/ammo_casing/spent
-	projectilesound = 'sound/weapons/Gunshot_light.ogg'
-	projectiletype = /obj/item/projectile/bullet/pistol/medium
-
-	weapon1 = /obj/item/weapon/gun/projectile/automatic/c20r
-
-/mob/living/simple_animal/hostile/syndicate/ranged/space
-	icon_state = "syndicaterangedpsace"
-	icon_living = "syndicaterangedpsace"
-	name = "Syndicate Commando"
-	min_oxy = 0
-	max_oxy = 0
-	min_tox = 0
-	max_tox = 0
-	min_co2 = 0
-	max_co2 = 0
-	min_n2 = 0
-	max_n2 = 0
-	minbodytemp = 0
-	corpse = /obj/effect/landmark/mobcorpse/syndicatecommando
-	speed = 0
-
-/mob/living/simple_animal/hostile/syndicate/ranged/space/Process_Spacemove(var/check_drift = 0)
-	return
-
-
-
-/mob/living/simple_animal/hostile/viscerator
-	name = "viscerator"
-	desc = "A small, twin-bladed machine capable of inflicting very deadly lacerations."
-	icon = 'icons/mob/critter.dmi'
-	icon_state = "viscerator_attack"
-	icon_living = "viscerator_attack"
-	pass_flags = PASSTABLE
-	health = 15
-	maxHealth = 15
-	melee_damage_lower = 15
-	melee_damage_upper = 15
-	attacktext = "cut"
-	attack_sound = 'sound/weapons/bladeslice.ogg'
-	faction = "syndicate"
-	min_oxy = 0
-	max_oxy = 0
-	min_tox = 0
-	max_tox = 0
-	min_co2 = 0
-	max_co2 = 0
-	min_n2 = 0
-	max_n2 = 0
-	minbodytemp = 0
-
-/mob/living/simple_animal/hostile/viscerator/death()
-	..(null,"is smashed into pieces!")
-	qdel(src)
+/mob/living/simple_animal/hostile/syndicate
+	name = "syndicate operative"
+	desc = "Death to the Company."
+	icon_state = "syndicate"
+	icon_living = "syndicate"
+	icon_dead = "syndicate_dead"
+	icon_gib = "syndicate_gib"
+
+	faction = "syndicate"
+	maxHealth = 100
+	health = 100
+	speed = 4
+
+	run_at_them = 0
+	cooperative = 1
+	investigates = 1
+	firing_lines = 1
+	returns_home = 1
+	reacts = 1
```

Comment:
If that's the case, then there are ~9 other cases where you can probably go back and remove the var(s) from other files.